m01_introduction/03_questionnaires.rst

```diff
@@ -189,14 +210,14 @@ else).
    students (83 %) have answered the question and most of them (300) have
    received 20 points. There is only one question whose correct answer is
    between 4.8 and 5.0, with 300 students giving that answer. Four students
-   have an incorrect answer having value between 1.4 and 1.6.
+   have an incorrect answer having a value between 1.4 and 1.6.
 
 
 .. figure:: /images/questionnaire/summary-freetext-string.png
   :alt: Screenshot of A+: summary of a string-type freetext question
 
   **Figure:** *the summary of a string-freetext questionnaire*. The answer
-  is an SQL query, and A+ shows the unique answers. There are four identical
+  is an SQL query, and A+ shows an unique answers. There are four identical
```

Comment:
"an unique answers" is wrong. It's either "the unique answers" or "unique answers".